maps/away/ships/coc/coc_scarab/coc_scarab.dm

```diff
@@ -203,13 +203,7 @@
 	name = "Burrow-class Freighter Engine Field Guide"
 	desc = "A paper."
 	info = "This is the field guide for the combustion engine of a Burrow-class freight vessel.<BR><BR>A combustion engine works by burning a fire in a combustion chamber, letting it burn out to 100% CO2, and then running the superheated gas through a vent, into a thermoelectric generator, and back into the chamber. This produces a lot of power, and will be required to keep the ship SMES topped up in the long term, but slowly cools down while in use. <BR><BR>GUIDE FOR USE:<BR><BR>Step 1: pressurise the cold loop with at least 2 canisters of hydrogen. The cold loop is the one that runs out into an external radiator array.<BR><BR>Step 2: configure the mixer to output a 60% oxygen and 40% hydrogen mix, and inject as much gas into the chamber as is available to you. <BR><BR>Step 3: cut injection, and ignite the mix. Do not panic when the glass makes a noise, that is normal and should stop after a short period.<BR><BR>Step 3: once the fire has fully burned out, enable combustion chamber injection at 700L/s and output at 1000kPa, so the gas begins to circulate through the thermoelectric generator. Lowering the output pressure will cause it to produce less power while causing the chamber to cool down slower, and raising the output pressure will cause the inverse. Do not generally raise the output above roughly 2000kPa, as this will cause it to cool down very quickly while generating more power than the engine SMES can actually intake, wasting heat. When you do not need power from it, disable the output and input to preserve the heat inside.<BR><BR>WARNING: Do not leave injection on after ignition, and if you think the glass might break, immediately cut fuel injection and lower the blast doors!"
-
-/obj/item/paper/fluff/scarabengine/Initialize()
-	. = ..()
-	var/languagetext = "\[lang=3\]"
-	languagetext += "[info]\[/lang\]"
-	info = parsepencode(languagetext)
-	icon_state = "paper_words"
```

Comment:
You removed the custom icon state on a lot of these, not sure if intentional. Paper_words should probably be set automatically if there's a language?

Reply:
Calling update_icon will set the icon state correctly if the paper has info